js/script.spec.js

```diff
@@ -34,5 +34,13 @@ describe("Beer display module", () => {
         const result = createBeerContainer().classList.contains('beerContainer');
         expect(result).toBeTruthy();
     });
+
+    test("createEndpoint without input returns expected text", () => {
+        expect(createEndpoint()).toBe("https://api.punkapi.com/v2/beers?page=1&per_page=15");
+    });
+
+    test("createEndpoint with input '&abv_lt=4' returns expected text", () => {
```

Comment:
Consider test name change to "createEndpoint with argument returns expected url"